README.md

```diff
@@ -25,9 +25,28 @@ then you just need to set deck default padding
 private val deckPager by lazy { activity.findViewById<Deck>(R.id.deck_pager) }
 
 private fun initDeck() {
-    deckPager.useDefaultPadding(activity)
+    deckPager.setDefaultPadding(activity)
```

Comment:
About this one, I think we don't really need it. We could force default padding if user of this library not specifying it. So it will be simpler for developer, 1 line gone :). What do you think?